pkg/templateservicebroker/servicebroker/provision.go

```diff
@@ -20,6 +23,12 @@ import (
 	"github.com/openshift/origin/pkg/templateservicebroker/util"
 )
 
+const (
+	invalidCustomLabelMsg = "Invalid Custom Label Parameter Value!"
+	missingLabelKeyMsg    = "Label Key Cannot Be Empty!"
+	invalidKeyValueStrMsg = "Key/Value String Cannot Contain Spaces!"
```

Comment:
if these are user facing I don't think you want to end them with exclamation points.

Reply:
I will update